lms/envs/aws.py

```diff
@@ -131,6 +131,10 @@
     if not STATIC_URL.endswith("/"):
         STATIC_URL += "/"
 
+# GITHUB_REPO_ROOT is the base directory
+# for course data
+GITHUB_REPO_ROOT = ENV_TOKENS.get('GITHUB_REPO_ROOT', GITHUB_REPO_ROOT)
```

Comment:
There's nothing specifically dependent on Github, is there? (As opposed to Bitbucket, Gitlab, etc) This name makes me sad.

Looks like we're already using it in the codebase, though, so I guess we're stuck with it. :cry: